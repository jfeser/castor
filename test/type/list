alist(select([f], r1) as k, ascalar(k.f))