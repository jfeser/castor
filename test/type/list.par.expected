(ListT ((IntT ((range (Interval 1 3)))) ((count (Interval 5 5)))))